aslam_offline_calibration/kalibr/scripts/kalibr_data_import.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+folder=$1
+cd $folder
+
+mkdir calibration_params && tar xzf calibrationdata.tar.gz -C ./calibration_params
+cd calibration_params/
+
+mkdir cam0
+for file in left-*.png; do
+  mv "$file" "./cam0/${file/left-/}"
+done
+
+mkdir cam1
+for file in right-*.png; do
+  mv "$file" "./cam1/${file/right-/}"
+done
+
+cd -
```

Comment:
This won't return to the original `pwd` (before calling the script). You need to save it on a variable and `cd` to it.

``` bash
OLDPWD=`pwd`

# bunch of code

cd "$OLDPWD"
```